docker-compose/technical/create-cassandra-databases.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
Do we have to add a license header for those scripts ?

Reply:
And the other files ?

Reply:
ok my bad